### PR TITLE
(chaostoolkit): Add csv, package and icons for chaostoolkit

### PR DESCRIPTION
**Pull request by chandankumar4**
Signed-off-by: Chandan Kumar <chandan.kr404@gmail.com>
This commit will contains the following:
- Add CSV file for chaostoolkit
- Add icons for k8-pod-delete and chaostoolkit
- Add package file

### CODE REVIEW

File: charts/chaostoolkit/experiments.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ description:
     Deletes a pod belonging to a deployment/statefulset/daemonset
 kind: ChaosExperiment
 metadata:
-  name: k8-pod-delete
+  name: k8s-pod-delete
```
Comment (ksatchit) [inline charts/chaostoolkit/experiments.yaml]: it is k8_pod_delete -->  they have a convention being used that way for other exp as well

File: charts/chaostoolkit/experiments.yaml [hunk 2]
```diff
@@ -58,7 +58,7 @@ spec:
       value: '50'
 
     labels:
-      name: k8-pod-delete
+      name: k8s-pod-delete
```
Comment (ksatchit) [inline charts/chaostoolkit/experiments.yaml]: same as above

File: charts/chaostoolkit/experiments.yaml (hunk 1 above)
Comment (chandankumar4) [reply charts/chaostoolkit/experiments.yaml]: okay!!

**Review by ajeshbaby — APPROVED** [review]
LGTM

**Review by ksatchit — APPROVED** [review]

**Pull request merged**